test/src/unit-cbor.cpp

```diff
@@ -1011,21 +1011,21 @@ TEST_CASE("CBOR")
                     SECTION("0 (0 00000 0000000000)")
                     {
                         json j = json::from_cbor(std::vector<uint8_t>({0xf9, 0x00, 0x00}));
-                        json::number_float_t d = j;
+                        json::number_float_t d{j};
```

Comment:
Is there a technical reason for this change, and if so, is this something that users of the library will have to do as well if they enable explicit conversions?

Reply:
I assume this is the same answer as the other one.  In that case, we should probably add something to the docs/README about the error that can happen and how to resolve it.

Reply:
The previous line simply does not compile when using explicit conversions. There are several ways to fix it, the easiest is the one I wrote, but you can be really explicit and use `get`, or even `static_cast`.